lib/poteto/review.rb

```diff
@@ -2,27 +2,28 @@
 
 module Poteto
   class Review
-    attr_accessor :path, :line, :comment
+    attr_accessor :path, :line, :comment, :commit_id
     attr_reader :raw_data
 
-    def initialize(raw_data, file_name: nil)
+    def initialize(raw_data, file_name: nil, commit_id: nil)
       @raw_data = raw_data
       @file_name = file_name
       @path = file_name unless file_name.nil?
+      @commit_id = commit_id unless commit_id.nil?
     end
   end
 
   class RubocopReview < Review
-    def initialize(raw_data, file_name: nil)
-      super(raw_data, file_name: file_name)
+    def initialize(raw_data, file_name: nil, commit_id: nil)
+      super(raw_data, file_name: file_name, commit_id: commit_id)
       parse
     end
 
     def parse
       path, @line, @comment  = @raw_data.split(",")
       @path ||= path.match(/file=(.*)$/)[1]
       @line = @line.match(/line=(\d+)/)[1].to_i
-      @comment = @comment.split("::").last
+      @comment = @comment.split("::")[2..]
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.